### ISSUE TITLE
git completions break when current working path contains spaces

### ISSUE THREAD

**Issue by deltaphi** [robbyrussell/oh-my-zsh#3227]
# Steps to reproduce
cd into a git repository
    mkdir "Directory With Plenty of Spaces"
    cd "Directory With Plenty of Spaces"
    touch modified-file.txt
    git add &lt;TAB&gt;

# Expected Result:
zsh completes the command line as `git add modified-file.txt``

# Actual Result:
My terminal rings the bell, complaining that there is no completion.

Do the same in "DirectoryWithoutAnySpaces" and the completion works as expected.

# System
OS X 10.9, System zsh
OhMyZsh Version: 96e4e5dd032d2df75f87d5b3e5594d527c655b16

**Comment by mcornella** [comment]
Could you figure it out?

**Comment by deltaphi** [comment]
I am still experiencing this issue. However, I haven't had time to follow up on this, yet.

**Comment by deltaphi** [comment]
I managed to look into it on my OSX machine again. I have two versions of ZSH installed:

```
$ /bin/zsh --version                                                      1 ↵
zsh 5.0.5 (x86_64-apple-darwin14.0)
$ /opt/local/bin/zsh --version
zsh 5.0.7 (x86_64-apple-darwin14.0.0)
````

The problem disappears when I forcefully use zsh 5.0.7 from MacPorts instead of the OSX-delivered version. Looking back through this thread, this issue was probably fixed with zsh 5.0.6.

**Comment by mcornella** [comment]
That's wonderful news! Thank you for the follow-up.
